lib/page/login_page.dart

```diff
@@ -56,8 +57,26 @@ class _LoginState extends State<_Login> {
               Padding(
                 padding: const EdgeInsets.all(8.0),
                 child: TextFormField(
-                  obscureText: true,
-                  decoration: const InputDecoration(hintText: "パスワード"),
+                  obscureText: _isDisplay,
+                  decoration: InputDecoration(
+                      suffixIcon: _isDisplay != true
+                          ? IconButton(
+                              icon: const Icon(Icons.visibility),
```

Comment:
[fix]
下記の方が綺麗になります〜
対応お願いします！
```suggestion
                              icon: Icon(_isDisplay != true ? Icons.visibility : Icons.visibility_off),
```